lib/src/browser_test_runner.dart

```diff
@@ -195,29 +195,3 @@ class BrowserTestRunner extends TestRunner {
            "${testFileName.replaceFirst(new RegExp(r"\.dart$"), ".html")}";
   }
 }
-
-
-/// Template of a Default HTML file for Browser unittest files that will
-/// start the tests in the Dart file written instead of the `{{test_file_name}}`
-/// placeholder.
-const String _BROWSER_TEST_HTML_FILE_TEMPLATE = '''
```

Comment:
It appears this was an unused duplicate of https://github.com/google/test_runner.dart/pull/35/files#diff-e64c28e5c61a6d30d00571548e89ee24R118